README.md

```diff
@@ -105,6 +105,7 @@ __fastify-gql__ supports the following options:
   exposed at `/graphql`.
 * `prefix`: String. Change the route prefix of the graphql endpoint if enabled.
 * `defineMutation`: Boolean. Add the empty Mutation definition if schema is not defined (Default: `false`).
+* `errorHandler`: Function. Change the default error handler (Default: `false`). _Note: If a custom error handler is defined, it should return the standardized response format according to [GraphQL spec](https://graphql.org/learn/serving-over-http/#response)._
```

Comment:
I think should be `default: true`, as there is a default error handler set.

Reply:
I think "`errorHandler`: Function. Change the default error handler (Default: `true`)" is quite ambiguous. Maybe rename the option to `customErrorHandler` and keep default `false` ?

Reply:
Based on you other comment I think that, for you, this option is not a function, but only a boolean that enable or disable fastify-gql handler.

Reply:
The default is `true`  at this point, as there is an error handler.